radixdlt/src/integration/java/com/radixdlt/consensus/deterministic/ControlledNetwork.java

```diff
@@ -242,7 +273,37 @@ public void highQC(QuorumCertificate qc) {
 		@Override
 		public void sendCommittedStateSync(long stateVersion, Object opaque) {
 			CommittedStateSync committedStateSync = new CommittedStateSync(stateVersion, opaque);
-			putMesssage(new ControlledMessage(sender, sender, committedStateSync));
+			putMessage(EARLIEST_POSSIBLE, new ControlledMessage(sender, sender, committedStateSync));
+		}
+
+		@Override
+		public void scheduleTimeout(LocalTimeout localTimeout, long milliseconds) {
+			putMessage(messageRank(localTimeout), new ControlledMessage(sender, sender, localTimeout));
+		}
+
+		private void putMessage(MessageRank eav, ControlledMessage controlledMessage) {
+			ControlledNetwork.this.messageQueue.add(eav, controlledMessage);
+		}
+
+		private MessageRank messageRank(EpochChange epochChange) {
```

Comment:
If we were to introduce a shared interface implemented by `EpochChange`, `NewView`, `Proposal`, `VertexMetaData` and `LocalTimeOut` that requires the existence of a `getEpoch()` method, then you can replace this 5 methods with the one single method? Feels neater? And might be beneficial in more places.

Reply:
Also not part of this PR.

Reply:
We could, but not sure that it makes sense to clutter up the production code to make the test code simpler.